nbs/core.ipynb

```diff
@@ -989,6 +1014,29 @@
     "        if isinstance(fcsts_df, pd.DataFrame) and _id_as_idx():\n",
     "            _warn_id_as_idx()\n",
     "            fcsts_df = fcsts_df.set_index(self.id_col)\n",
+    "\n",
+    "        # perform conformal predictions\n",
+    "        if conformal_level is not None:\n",
```

Comment:
I think we should just call this `level`, that will also generalize it when we want to use the argument to generate arbitrary levels with DistributionLoss functions.